etc/validation-error-message-formatter.js

```diff
@@ -1,4 +1,49 @@
+exports.hashtableMaximumSizeViolation = deprecate(errorFormatter.hashtableMaximumSizeViolation, deprecationMessage);
+exports.hashtableMinimumSizeViolation = deprecate(errorFormatter.hashtableMinimumSizeViolation, deprecationMessage);
+exports.immutableDocViolation = deprecate(errorFormatter.immutableDocViolation, deprecationMessage);
+exports.immutableItemViolation = deprecate(errorFormatter.immutableItemViolation, deprecationMessage);
+exports.maximumAttachmentCountViolation = deprecate(errorFormatter.maximumAttachmentCountViolation, deprecationMessage);
+exports.maximumIndividualAttachmentSizeViolation =
+  deprecate(errorFormatter.maximumIndividualAttachmentSizeViolation, deprecationMessage);
+exports.maximumLengthViolation = deprecate(errorFormatter.maximumLengthViolation, deprecationMessage);
+exports.maximumSizeAttachmentViolation = deprecate(errorFormatter.maximumSizeAttachmentViolation, deprecationMessage);
+exports.maximumTotalAttachmentSizeViolation = deprecate(errorFormatter.maximumTotalAttachmentSizeViolation, deprecationMessage);
+exports.maximumValueViolation = deprecate(errorFormatter.maximumValueViolation, deprecationMessage);
+exports.maximumValueExclusiveViolation = deprecate(errorFormatter.maximumValueExclusiveViolation, deprecationMessage);
+exports.minimumLengthViolation = deprecate(errorFormatter.minimumLengthViolation, deprecationMessage);
+exports.minimumValueViolation = deprecate(errorFormatter.minimumValueViolation, deprecationMessage);
+exports.minimumValueExclusiveViolation = deprecate(errorFormatter.minimumValueExclusiveViolation, deprecationMessage);
+exports.mustNotBeEmptyViolation = deprecate(errorFormatter.mustNotBeEmptyViolation, deprecationMessage);
+exports.regexPatternHashtableKeyViolation = deprecate(errorFormatter.regexPatternHashtableKeyViolation, deprecationMessage);
+exports.regexPatternItemViolation = deprecate(errorFormatter.regexPatternItemViolation, deprecationMessage);
+exports.requireAttachmentReferencesViolation = deprecate(errorFormatter.requireAttachmentReferencesViolation, deprecationMessage);
+exports.requiredValueViolation = deprecate(errorFormatter.requiredValueViolation, deprecationMessage);
+exports.supportedContentTypesAttachmentReferenceViolation =
+  deprecate(errorFormatter.supportedContentTypesAttachmentReferenceViolation, deprecationMessage);
+exports.supportedContentTypesAttachmentViolation =
+  deprecate(errorFormatter.supportedContentTypesAttachmentReferenceViolation, deprecationMessage);
+exports.supportedContentTypesRawAttachmentViolation =
+  deprecate(errorFormatter.supportedContentTypesRawAttachmentViolation, deprecationMessage);
+exports.supportedExtensionsAttachmentReferenceViolation =
+  deprecate(errorFormatter.supportedExtensionsAttachmentReferenceViolation, deprecationMessage);
+exports.supportedExtensionsAttachmentViolation =
+  deprecate(errorFormatter.supportedExtensionsAttachmentReferenceViolation, deprecationMessage);
```

Comment:
Notice that, because the `supportedExtensionsAttachmentViolation` function no longer exists in `src/validation-error-message-formatter.js`, I have wired it up to `errorFormatter.supportedExtensionsAttachmentReferenceViolation` for backward compatibility directly in the deprecated version.